File: python/6kyu/Split_Strings.py
def solution(s):
    if len(s) % 2 == 1:
        s = s + "_"
        
    res = []
    
    #print(s[0:2])
    
    while len(s) > 0:
        res.append(s[0:2])
        s = s[2:]
        
    return res

File: csharp/6kyu/Split_Strings.cs
using System;
using System.Collections.Generic;

public class SplitString
{
  public static string[] Solution(string str)
  {
    List<string> lStr = new List<string>();
    string tStr = null;
    
    for(int i = 0; i < str.Length; i += 2){
      if(str.Length >= i + 3){
        tStr = str.Substring(i, 2);
      }else{
        tStr = str.Substring(i);
      }
      if(tStr.Length == 1){
        tStr = tStr + '_';
      }
      lStr.Add(tStr);
    }
    
    
    return lStr.ToArray();
  }
}